Packages/KYAApplicationSupport/Sources/KYAApplicationSupport/include/KYAApplicationSupport/NSUserDefaults+Keys.h

```diff
@@ -57,6 +58,12 @@ FOUNDATION_EXPORT NSString * const KYAUserDefaultsKeyPreReleaseUpdatesEnabled;
  */
 @property (nonatomic, getter=kya_isQuitOnTimerExpirationEnabled) BOOL kya_quitOnTimerExpirationEnabled;
 
+/**
+ Returns YES if the app should activate when external display is connected.
+ */
```

Comment:
For doc blocks I tend to use the `/// ` style for new code. I would prefer the same here.
```
/// Returns YES…
```